packages/bruno-electron/src/ipc/network/interpolate-vars.js

```diff
@@ -28,6 +28,35 @@ const interpolateEnvVars = (str, processEnvVars) => {
   });
 };
 
+// Updated interpolate function to handle nested variables
+const interpolate = (str, combinedVars) => {
+  if (!str || !str.length || typeof str !== 'string') {
+    return str;
+  }
+
+  // Replace nested variables like {{auth.token}} all other types
```

Comment:
@razzivofficial Currently we don't plan to support nested variables (at least in v1) `auth.token` is not a valid variable name.